src/schema/sources.ts

```diff
@@ -934,7 +950,11 @@ export const canPostToSquad = (
     return false;
   }
 
-  return sourceRoleRank[sourceMember.role] >= squad.memberPostingRank;
+  const memberRank = sourceRoleRank[sourceMember.role];
+
+  if (memberRank >= sourceRoleRank.moderator) return true;
+
+  return !squad.moderationRequired && memberRank >= squad.memberPostingRank;
```

Comment:
Just sanity check if this will work for current squads for backward compatibility? `moderationRequired` is false by default?

Reply:
Haha, yep! It already caused a problem 😅

https://github.com/dailydotdev/daily-api/pull/2324